scripts/tests/test_encoder_decoder.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 import mxnet as mx
 from mxnet.test_utils import assert_almost_equal
-from ..nmt.encoder_decoder import *
+from ..nmt.gnmt_encoder_decoder import *
+from ..nmt.transformer_encoder_decoder import *
```

Comment:
let's have the following module names:
nmt.encoder_decoder: defines the encoder decoder and shared functionality
nmt.gnmt: everything in gnmt including the encoder and decoder
nmt.transformer: everything in transformer.

Reply:
How about the training scripts? we have gnmt.py and transformer.py

Reply:
I was considering using train_gnmt.py and train_transformer.py. However, our other scripts do not use 'train' in the script name. So I keep using these to ensure consistency.

Reply:
train_ prefix sounds good